ethereum/client/src/tests.rs

```diff
@@ -0,0 +1,247 @@
+use alloy::{hex::FromHex, sol};
+
+sol!(
+    #[sol(rpc)]
+    ProxyContract,
+    "../out/ProxyContract.sol/ProxyContract.json"
+);
+
+sol!(
+    #[sol(rpc)]
+    ERC20Mock,
+    "../out/ERC20Mock.sol/ERC20Mock.json"
+);
+
+sol!(
+    #[sol(rpc)]
+    Relayer,
+    "../out/Relayer.sol/Relayer.json"
+);
+
+sol!(
+    #[sol(rpc)]
+    MessageQueue,
+    "../out/MessageQueue.sol/MessageQueue.json"
+);
+
+sol!(
+    #[sol(rpc)]
+    Verifier,
+    "../out/Verifier.sol/Verifier.json"
+);
+
+#[cfg(test)]
```

Comment:
You don't need `#[cfg(test)]` anywhere in this file

Reply:
Fixed